README.md

```diff
@@ -141,15 +143,32 @@ And then execute:
 $ bundle install
 ```
 
-Finally, add these lines to your routes.rb:
+### Update routes file
+
+Add these lines to your routes.rb:
 
 ```ruby
+# config/routes.rb in your main Rails app
```

Comment:
I thought it could be useful to clarify main vs. driver routes file, but not sure if it's necessary.